handlers/roaster_account.go

```diff
@@ -19,74 +20,77 @@ type RoasterAccountIfc interface {
 }
 
 type RoasterAccount struct {
-	sql *gateways.Sql
+	Helper *helpers.RoasterAccount
 }
 
-func NewRoasterAccount(sql *gateways.Sql) RoasterAccountIfc {
-	return &RoasterAccount{sql: sql}
+func NewRoasterAccount(sql gateways.SQL) RoasterAccountI {
+	return &RoasterAccount{Helper: helpers.NewRoasterAccount(sql)}
 }
 
 func (c *RoasterAccount) New(ctx *gin.Context) {
-	var json containers.RoasterAccount
+	var json models.RoasterAccount
 	err := ctx.BindJSON(&json)
 
 	if err != nil {
 		ctx.JSON(400, errResponse("Invalid RoasterAccount Object"))
-		fmt.Printf("%s",err.Error())
+		fmt.Printf("%s", err.Error())
 		return
 	}
 
-  // TODO:  Update parameters
-	err = c.sql.Modify(
-		"INSERT INTO roaster_account VALUE(?)",
-		uuid.New())
+	// TODO: create stripe account?
```

Comment:
dope. i think figuring out what we do when creating/updating/deleting accounts will be the semi-complex part here.